omni_dior.mk

```diff
@@ -27,12 +27,12 @@ PRODUCT_COPY_FILES += \
     bionic/libc/zoneinfo/tzdata:recovery/root/system/usr/share/zoneinfo/tzdata
 
 PRODUCT_COPY_FILES += \
-    device/xiaomi/armani/kernel:kernel \
-    device/xiaomi/armani/dt.img:dt.img
+    device/xiaomi/dior/kernel:kernel \
+    device/xiaomi/dior/dt.img:dt.img
```

Comment:
We don't need this, line 29-31 can be removed. It's already defined at BoardConfig.mk